Comparing source compatibility of /root/workspace/apache/flink/383515229/flink-streaming-java/target/flink-streaming-java_2.11-1.6-SNAPSHOT.jar against /root/./workspace/apache/flink/383515229/.m2/org/apache/flink/flink-streaming-java_2.11/1.4.0/flink-streaming-java_2.11-1.4.0.jar
***  MODIFIED CLASS: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.streaming.api.datastream.SplitStream split(org.apache.flink.streaming.api.collector.selector.OutputSelector)
***  MODIFIED CLASS: PUBLIC org.apache.flink.streaming.api.environment.CheckpointConfig  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PUBLIC(+) boolean isFailOnCheckpointingErrors()
	+++  NEW METHOD: PUBLIC(+) void setFailOnCheckpointingErrors(boolean)
+++  NEW CLASS: PUBLIC(+) org.apache.flink.streaming.api.environment.LegacyLocalStreamEnvironment  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- -1.-1
	+++  NEW SUPERCLASS: org.apache.flink.streaming.api.environment.LocalStreamEnvironment
	+++  NEW CONSTRUCTOR: PUBLIC(+) LegacyLocalStreamEnvironment()
	+++  NEW CONSTRUCTOR: PUBLIC(+) LegacyLocalStreamEnvironment(org.apache.flink.configuration.Configuration)
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.api.common.JobExecutionResult execute(java.lang.String)
		+++  NEW EXCEPTION: java.lang.Exception
